Source/p_mobj.h

```diff
@@ -336,6 +336,10 @@ typedef struct mobj_s
 
     // killough 8/2/98: friction properties part of sectors,
     // not objects -- removed friction properties from here
+    // e6y: restored friction properties here
```

Comment:
Will this change break savegames?

Reply:
Yes, it breaks savegame. Not sure how to deal with it.

Reply:
This really nags me...

Reply:
Yes, I think this is the last issue I tested about 5000 complevel 9 demos without desync.